reactive/kubernetes_control_plane.py

```diff
@@ -1809,60 +1808,7 @@ def addons_ready():
         return False
 
 
-@when("ceph-storage.available")
-def ceph_state_control():
```

Comment:
Could you also `block` on this hook to inform folks not to use this relation?  In the case of an upgrade to this charm -- i think i'd wanna know why this relation isn't doing what it used to

Reply:
also, are there any flags that can be removed/cleared if the `ceph-storage` relation is disconnected

Reply:
> Could you also `block` on this hook to inform folks not to use this relation? In the case of an upgrade to this charm -- i think i'd wanna know why this relation isn't doing what it used to

Good idea. I've added a blocked status message for it.

> also, are there any flags that can be removed/cleared if the ceph-storage relation is disconnected

I don't think so. It looks like the only relevant flag is `ceph-storage.available` but the ceph-admin interface clears it on departure [here](https://github.com/openstack-charmers/juju-interface-ceph-admin/blob/c1061a29297084fa53c2474ba371671186ff3389/requires.py#L6-L11).